src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/DistributionScheduler.java

```diff
@@ -24,6 +24,10 @@ public DistributionScheduler(ActionDistributor actionDistributor) {
   /** Scheduled execution of the Action Distributor */
   @Scheduled(fixedDelayString = "#{appConfig.actionDistribution.delayMilliSeconds}")
   public void run() {
-    actionDistributor.distribute();
+    try {
+      actionDistributor.distribute();
+    } catch (Exception ex) {
```

Comment:
Having had a quick look I think the answer is no, but can we be more specific than `Exception` here?

Reply:
Reason for catch-all is that we want to know what kinds of runtime exceptions and things are bubbling up all the way to the top here.